tortoise/backends/base/executor.py

```diff
@@ -264,16 +265,20 @@ def get_update_sql(
     async def execute_update(
         self, instance: "Union[type[Model], Model]", update_fields: Optional[Iterable[str]]
     ) -> int:
+        fields: "dict[str, Field]" = {
```

Comment:
We don't have performance benchmarks for updates but I'm pretty sure that two loops is slower than a single loop. Why not just check for the pk field in the existing loop?